docs/usage.rst

```diff
@@ -94,22 +100,27 @@ You can also customize the template directory from a test by passing in the opti
 
    @pytest.fixture
    def custom_template(tmp_path) -> Path:
+       # Create custom copier template directory and copier.yaml file
+       (template := tmp_path / "copier-template").mkdir()
+       questions = {"custom_name": {"type": "str", "default": "my_default_name"}}
+       (template /"copier.yaml").write_text(yaml.dump(questions))
+       # Create custom subdirectory
+       (repo_dir := template / "custom_template").mkdir()
+       (template /"copier.yaml").write_text(yaml.dump({"_subdirectory": "custom_template"}))
```

Comment:
I found when I tried to include `_subdirectory` in the questions dictionary, the yaml.dump would put them in alphabetical order, i.e. list `_subdirectory` first in `copier.yaml`. That seemed to make copier unhappy.